great-idea/services.html

```diff
@@ -16,91 +16,116 @@
 
 <body>
```

Comment:
recommend creating a div that encompasses everything inside the body with a class called `layout` or `container`.

- body
  - container
    - header
      - nav
    - main
      - main components of page such as div, etc.

Reply:
```
<body>
  <div class="page-name">
    <header>
      Gecko facts
      
      <nav>alsdfj</nav>
    </header>

    <main role="main">
      <div>
        <p>Geckos are a group of usually small, usually nocturnal lizards. They are found on every continent except Australia.</p>
      </div>
        

        <p>Many species of gecko have adhesive toe pads which enable them to climb walls and even windows.</p>
    </main>
  </div>
</body>
```